lib/config/react.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  'extends': ['./base', 'tslint-react'],
```

Comment:
This is how it's specified in their [usage documentation](https://github.com/palantir/tslint-react#usage)
I presume it helps tslint know that we want to use tslint-react and then we override the preconfigured tslint-react rules.